<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Room</class>
 <widget class="QMainWindow" name="Room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_room">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入房间号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Join">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>请求加入</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_room">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Ready">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>准备</string>
    </property>
   </widget>
   <widget class="QLabel" name="Player1_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player1_state">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player2_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player2_state">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player4_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player3_state">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player4_state">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>390</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player3_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player6_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player5_state">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player6_state">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player5_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Port">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入端口号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>19</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入服务器地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Host">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>尝试连接</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
